rct229/rulesets/ashrae9012019/section19/section19rule36.py

```diff
@@ -0,0 +1,94 @@
+            )
+
+        def get_calc_vals(self, context, data=None):
+            fan_sys_b = context.baseline
+
+            sensible_eff_b = find_one(
+                "$.air_energy_recovery.design_sensible_effectiveness", fan_sys_b
+            )
+            latent_eff_b = find_one(
+                "$.air_energy_recovery.design_latent_effectiveness", fan_sys_b
+            )
+            ERV_OA_airflow_b = find_one(
+                "$.air_energy_recovery.outside_air_flow", fan_sys_b
+            )
+            ERV_EA_airflow_b = find_one(
+                "$.air_energy_recovery.exhaust_air_flow", fan_sys_b
+            )
+            hvac_min_oa_flow_b = find_one("$.minimum_outdoor_airflow", fan_sys_b)
+
+            return {
+                "sensible_eff_b": sensible_eff_b,
+                "latent_eff_b": latent_eff_b,
+                "ERV_OA_airflow_b": CalcQ("air_flow_rate", ERV_OA_airflow_b),
+                "ERV_EA_airflow_b": CalcQ("air_flow_rate", ERV_EA_airflow_b),
+                "hvac_min_oa_flow_b": CalcQ("air_flow_rate", hvac_min_oa_flow_b),
+            }
+
+        def applicability_check(self, context, calc_vals, data):
+            fan_sys_b = context.baseline
+
+            sensible_eff_b = calc_vals["sensible_eff_b"]
+            latent_eff_b = calc_vals["latent_eff_b"]
+            ERV_OA_airflow_b = calc_vals["ERV_OA_airflow_b"]
+            ERV_EA_airflow_b = calc_vals["ERV_EA_airflow_b"]
+            hvac_min_oa_flow_b = calc_vals["hvac_min_oa_flow_b"]
+
+            return (
+                fan_sys_b.get("air_energy_recovery") is not None
+                and fan_sys_b["air_energy_recovery"] != ENERGY_RECOVERY.NONE
+                and std_equal(REQ_SENSIBLE_EFFECTIVENESS, sensible_eff_b)
+                and std_equal(REQ_LATENT_EFFECTIVENESS, latent_eff_b)
+                and std_equal(hvac_min_oa_flow_b, ERV_OA_airflow_b)
+                and std_equal(hvac_min_oa_flow_b, ERV_EA_airflow_b)
+            )
+
+        def get_manual_check_required_msg(self, context, calc_vals=None, data=None):
+            hvac_b = context.baseline
+            hvac_id_b = hvac_b["id"]
+
+            sensible_eff_b = calc_vals["sensible_eff_b"]
+            latent_eff_b = calc_vals["latent_eff_b"]
+            ERV_OA_airflow_b = calc_vals["ERV_OA_airflow_b"]
+            ERV_EA_airflow_b = calc_vals["ERV_EA_airflow_b"]
+            hvac_min_oa_flow_b = calc_vals["hvac_min_oa_flow_b"]
+
+            return (
+                f"{hvac_id_b} was modeled with a design sensible effectiveness of {sensible_eff_b}, a design latent effectiveness of {latent_eff_b}, and with {ERV_OA_airflow_b.to(ureg.cfm).m} outdoor air cfm and with {ERV_EA_airflow_b.to(ureg.cfm).m} exhaust air cfm going through the energy recovery device. "
+                f"The HVAC system's minimum outside air CFM is {hvac_min_oa_flow_b.to(ureg.cfm).m}. Verify that it is equivalent to 50% enthalpy recovery ratio required by G3.1.2.10."
```

Comment:
```suggestion
                f"The HVAC system's minimum outside air CFM is {hvac_min_oa_flow_b.to(ureg.cfm).m}. Verify that it is equivalent to 50% enthalpy recovery ratio required by G3.1.2.10."
```

What does `hvac_min_oa_flow_b.to(ureg.cfm)` do?
@yunjoonjung-PNNL

Reply:
It changes the ```hvac_min_oa_flow_b```'s unit to cfm. Since the ```pint``` package is used, we don't need to worry about units when we compare two quantities. 

However, in this case, I converted to cfm because the message specified the ```hvac_min_oa_flow_b``` quantity in cfm.